### PR TITLE
button log and devise

### PR DESCRIPTION
**Pull request by anthomanto**
Qui valide mon joli bouton

### CODE REVIEW

**Review by MalgouyreBGA — APPROVED** [review]

File: app/views/shared/_navbar.html.erb [hunk 1]
```diff
@@ -1,34 +1,35 @@
 <div class="navbar navbar-expand-sm navbar-light navbar-lewagon">
   <%= link_to "#", class: "navbar-brand ml-3" do %>
-    <%= image_tag "https://image.flaticon.com/icons/svg/1055/1055646.svg" %>
-    <% end %>
-
+  <%= image_tag "https://image.flaticon.com/icons/svg/1055/1055646.svg" %>
```
Comment (MaximeAlexandre) [inline app/views/shared/_navbar.html.erb]: indent please

File: app/views/shared/_navbar.html.erb [hunk 2]
```diff
@@ -1,34 +1,35 @@
...
+  <% end %>
   <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
-    <span class="navbar-toggler-icon"></span>
+  <span class="navbar-toggler-icon"></span>
```
Comment (MaximeAlexandre) [inline app/views/shared/_navbar.html.erb]: indent please

File: app/views/shared/_navbar.html.erb [hunk 3]
```diff
@@ -1,34 +1,35 @@
...
+        <%= link_to "Messages", "#", class: "nav-link" %>
+      </li>
+      <li class="nav-item dropdown">
+        <%= image_tag "https://kitt.lewagon.com/placeholder/users/ssaunier", class: "avatar dropdown-toggle", id: "navbarDropdown", data: { toggle: "dropdown" }, 'aria-haspopup': true, 'aria-expanded': false %>
+        <div class="dropdown-menu dropdown-menu-right" aria-labelledby="navbarDropdown">
+          <%= link_to "Action", "#", class: "dropdown-item" %>
+          <%= link_to "Another action", "#", class: "dropdown-item" %>
+          <%= link_to "Log out", destroy_user_session_path, method: :delete, class: "dropdown-item" %>
+        </div>
+      </li>
```
Comment (MaximeAlexandre) [inline app/views/shared/_navbar.html.erb]: indent line 11 to 24

File: app/views/shared/_navbar.html.erb [hunk 4]
```diff
@@ -1,34 +1,35 @@
...
-        <li class="nav-item mr-3">
-          <%= link_to "Login", new_user_session_path, class: "nav-link", :style=>'color: white;'%>
-        </li>
+      <li class="nav-item dropdown mr-3" href="#" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+        <%= link_to "Join the Spaceship", new_user_session_path, class: "nav-link dropdown-toggle", :style=>'color: white;'%>
+      </li>
+      <div class="dropdown-menu" aria-labelledby="navbarDropdown">
+        <%= link_to "Login", new_user_session_path, class: "nav-link2" , :style=> ' border-bottom: solid 1px #262626;'%>
+        <%= link_to "Sign up", new_user_registration_path, class: "nav-link2"%>
+      </div>
```
Comment (MaximeAlexandre) [inline app/views/shared/_navbar.html.erb]: indent line 26 to 32

**Review by MaximeAlexandre — CHANGES_REQUESTED** [review]
Check indent and push again but not in MASTER xD

**Review by MaximeAlexandre — APPROVED** [review]

**Pull request merged**